data/tutorials/getting-started/2_00_editor_setup.md

```diff
@@ -49,10 +49,74 @@ If you used the DkML distribution, you will need to:
     1. Go to `File` > `Preferences` > `Settings` view (or press `Ctrl ,`)
     2. Select `User` > `Extensions` > `OCaml Platform`
     3. Uncheck `OCaml: Use OCaml Env`. That's it!
+    
+## Emacs
 
-## Vim and Emacs
+Using Emacs to work with OCaml requires at least two modes:
 
-**For Vim and Emacs**, we won't use the LSP server but rather directly talk to Merlin.
+- A major mode, which, among other things, supports syntax highlighting and the structuring of indentation levels
+- A minor mode, which will interact with a language server (such as `ocaml-lsp-server` or `merlin`). In this tutorial, we will focus on using the new `ocaml-eglot` mode and `ocaml-lsp-server` as a server.
+
+### Choosing a major mode
+
+There are several major modes dedicated to OCaml, of which the 3 main ones are:
+
+- [Tuareg](https://github.com/ocaml/tuareg): an old-fashioned (but still updated), very complete mode, usually the recommended one
+- [Caml](https://github.com/ocaml/caml-mode): a mode even older than `tuareg` (but still updated), lighter than `tuareg`
+- [Neocaml](https://github.com/bbatsov/neocaml): a brand new mode, based on modern tools (like [tree-sitter](https://tree-sitter.github.io/tree-sitter/)). Still experimental at the time of writing.
+
+For the purposes of this tutorial, we are going to focus on the use of `tuareg` as the major mode, but you should feel free to experiment and choose your favourite one! To use `tuareg`, you can add these lines to your Emacs configuration:
+
+```elisp
+(use-package tuareg
+  :ensure t
+  :mode (("\\.ocamlinit\\'" . tuareg-mode)))
+```
+
+> If your version of Emacs does not support the `use-package` macro (or is not set up to take MELPA packages into account), please update it and follow these instructions to install [`use-package`](https://github.com/jwiegley/use-package) and [MELPA](https://melpa.org/#/getting-started).
```

Comment:
Quotes should be used for quotes, not highlighting. We usually have paragraph header, in bold, such as “remark”, “fun fact”, “warning”, “note” or something else.

Reply:
Should be fixed on https://github.com/ocaml/ocaml.org/pull/3064/commits/c1bb0be2592f67472a775588cf7a528ca23bd561